src/cascadia/TerminalApp/CommandPalette.idl

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import "../Command.idl";
+
+namespace TerminalApp
+{
+    [default_interface] runtimeclass CommandPalette : Windows.UI.Xaml.Controls.Grid
+    {
+        CommandPalette();
+
+        Windows.Foundation.Collections.IObservableVector<Command> FilteredActions { get; };
+
+        void SetActions(Windows.Foundation.Collections.IVector<Command> actions);
```

Comment:
I'm wondering if we need an "action provider" model moreso than an action push model. Unless that vector is shared? Maybe we need to follow tabview's lead and do a `SetActionSource`?

Reply:
Or, `ActionsSource {set; get;}`. bind w/ xaml.